src/storage.js

```diff
@@ -0,0 +1,5 @@
+export default class StorageManager {
+  static save = (tasks) => {
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  };
+}
```

Comment:
- [ ] Your function is saving changes when clicking clearCompleted & deleting tasks to localStorage but when you reload the page none of the data is persisted. I suspect because you are not loading that data and displaying it on screen. 
- [ ] please kindly also ensure when updating the task like deleting. the display is refreshed to show the new state of the tasks.

Reply:
Updated, 
thank you.